### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by allurivamsi5**

### CODE REVIEW

**Comment by allurivamsi5** [comment]
Anna can you review now..

File: templates/registrationPage.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<html>
+    <head>
+        <title>Registerpage!</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+        <link rel = "stylesheet" href="{{ url_for('static',filename='registrationPage.css') }}">
+        <!-- <link rel="stylesheet" href="registrationPage.css"> -->
```
Comment (ashishk144) [inline templates/registrationPage.html:11]: remove commented code.

File: Project1_Notes.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Task-1
```
Comment (ashishk144) [inline Project1_Notes.md:1]: Add task time.

File: application.py [hunk 3]
```diff
@@ -23,4 +23,17 @@
 
 @app.route("/")
```
Comment (ashishk144) [inline application.py:24]: redirect it to /register.

**Comment by allurivamsi5** [comment]
Anna, updated as per the feedback...once check...

**Comment by allurivamsi5** [comment]
Bro, updated Userdatabse three tasks, once check and review .

**Comment by ashishk144** [comment]
> Bro, updated Userdatabse three tasks, once check and review .

Can you clarify on which python file you're using application.py or application_1.py?

**Comment by allurivamsi5** [comment]
Bro, application_1.py file used

**Pull request merged**